### ISSUE TITLE
EDA segfaults on Ubuntu 22.10

### ISSUE THREAD

**Issue by rozyczko** [easyScience/easyDiffractionApp#216]

**Comment by rozyczko** [comment]
It seems the same happens for 22.04 LTS

**Comment by rozyczko** [comment]
Building EDA on 22.04 (GH run) makes it run properly on 22.* apart of the known issue of not visualising the charts and structure.

**Comment by mducle** [comment]
:bump: I've just encountered this issue. I checked `0.8.2`, `0.8.3`, and `0.8.4` and they all give segfaults. 

Is there a way to install the app using `conda` or `pip`?

**Comment by rozyczko** [comment]
> Is there a way to install the app using `conda` or `pip`?

Not really. They both target <=20.04 only. If you are adventurous, you could try installing it with poetry, with `poetry install` in the main EDA **source** directory (git cloned).

For 0.8.5, due at some point this quarter, we will target 22.x as well.

**Comment by mducle** [comment]
Ok. Thanks. I'll try that.

Also, I tried using the windows version under `wine` but although it starts up, it crashes when you try to load a project.

**Comment by rozyczko** [comment]
> Also, I tried using the windows version under `wine` but although it starts up, it crashes when you try to load a project.

Which release? 0.8.4? Which project did you try to load?

**Comment by mducle** [comment]
> Which release? 0.8.4? Which project did you try to load?

It was release 0.8.4 running on Wine 7.2. The crash happens for every example. The stack trace is [here](https://gist.github.com/mducle/ce535a8190c63f83d0577f1c695ee0d9). 

If you create your own project you can load a `cif` file with the model ok, but then it crashes when you try to load an experimental data (`xye`) file.

**Comment by rozyczko** [comment]
Simon hopefully fixed this by using `snap` to package Linux installers and things should be Ubuntu version agnostic now.
This should go into v. 0.8.5, available soon®
